src/components/Collection.jsx

```diff
@@ -1,7 +1,17 @@
 import React from "react";
-
+import Card from "../containers/card/Card";
+import { useSelector } from "react-redux";
 function Collection() {
-  return <div>Collection</div>;
+  const collectionItems = useSelector(
+    (state) => state.сollectionItems.сollectionItems
```

Comment:
селекторы выносим в отдельный файл

Reply:
```suggestion
   const collectionItems = useSelector(colectionSelector);
```

в файле с селектором:
 export const colectionSelector = (state) => state.сollectionItems.сollectionItems